yellowbrick/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "yellowbrick" %}
+{% set version = "1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3edddb4a16ddd0586169fccd0645b2fa78e266c40968f872753909250ac6826c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

I think this can be ``noarch``
```suggestion
  noarch: python
```

---

Removing the script command made the build fail, but I'll leave noarch